eslint.config.js

```diff
@@ -0,0 +1,222 @@
+import js from '@eslint/js';
```

Comment:
Since this is JS file it needs to follow the Google's JS style guide:
https://google.github.io/styleguide/jsguide.html

Generally looks good, but I see some lines to be column wrapped at 80.